engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -95,7 +95,9 @@
 public class StramClient
 {
   private static final Logger LOG = LoggerFactory.getLogger(StramClient.class);
-  public static final String YARN_APPLICATION_TYPE = "DataTorrent";
+  public static final String YARN_APPLICATION_TYPE_DEPRECATED = "DataTorrent";
+  public static final String YARN_APPLICATION_TYPE = "ApacheApex";
```

Comment:
Have you checked what other Apache projects are using? I'm fine with ApacheApex and I suspect it's inline with the ASF branding guidelines.